docs/self-hosted-getting-started.md

```diff
@@ -0,0 +1,99 @@
+# Quick Start: Run a self-hosted database instance with automatic vectorization
+
+If you want to see how it works, follow these steps to setup 
+a self-hosted database instance with automatic embedding vectorization:
+
+## Pre-requisites
+- Docker installed on your machine
+- An OpenAI API key. You can create one [here](https://platform.openai.com/api-keys).
+
+## Setup
+
+1. Run the following docker-compose file to start a TimescaleDB instance and the Vectorizer worker: 
+    ```yaml
+    services:
+      db:
+        image: timescale/timescaledb-ha:cicd-024349a-arm64
+        environment:
+          POSTGRES_PASSWORD: postgres
+          OPENAI_API_KEY: your-api-key
+        ports:
+          - "5432:5432"
+        volumes:
+          - ./data:/var/lib/postgresql/data
+    
+      vectorizer:
+        image: timescale/pgai-vectorizer-worker:rc
```

Comment:
We only have available `timescale/pgai-vectorizer-worker:0.1.0rc4`

Reply:
oh yes somehow I must have messed this up while copy pasting back and forth 😅